src/main/java/org/itsallcode/openfasttrace/importer/legacytag/config/DescribedPathMatcher.java

```diff
@@ -0,0 +1,93 @@
+package org.itsallcode.openfasttrace.importer.legacytag.config;
+
+/*-
+ * #%L
+ * OpenFastTrace
+ * %%
+ * Copyright (C) 2016 - 2018 itsallcode.org
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * #L%
+ */
+
```

Comment:
Please add JavaDoc to all public methods. To package-scoped optionally - but recommended.

Reply:
This is only used internally. Users only work with `PathConfig`.

Reply:
:heavy_check_mark: